.github/actions/install-chrome/action.yml

```diff
@@ -0,0 +1,29 @@
+name: Install Chrome browser and driver for Testing
+description: Download and install the latest available Chrome for Testing and Chromedriver
+
+runs:
+  using: composite
+  steps:
+    - id: cache-chrome-browser
+      name: Chrome browser cache
+      uses: actions/cache@v3
+      with:
+        path: ./chrome/**/chrome-linux64/chrome
+        key: chrome
+
+    - id: cache-chromedriver
+      name: Chrome driver cache
+      uses: actions/cache@v3
+      with:
+        path: ./chromedriver/**/chromedriver-linux64/chromedriver
+        key: chromedriver
+
+    - id: install-chrome
+      name: Install Chrome
+      shell: bash
+      run: |
+        sudo apt-get remove google-chrome-stable
+        npx @puppeteer/browsers install chrome
+        npx @puppeteer/browsers install chromedriver
+        sudo ln -fs ./chrome/**/chrome-linux64/chrome /usr/bin/chrome
+        sudo ln -fs ./chromedriver/**/chromedriver-linux64/chromedriver $CHROMEWEBDRIVER/chromedriver
```

Comment:
Probably this will work fine, but if multiple versions of `chrome` or `chromedriver` are present (e.g. restored from cache), which version will be linked?